code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm

```diff
@@ -374,4 +374,32 @@
 	. = ..()
 	qdel(vat)
 
+//randomizes from the netherworld pool!
+/datum/micro_organism/cell_line/netherworld
+	desc = "Aberrant residue"
+	required_reagents = list(//theme here: very odd requirements
+							/datum/reagent/water/hollowwater,//geyser reagent, so plentiful when found
+							/datum/reagent/consumable/ethanol/wizz_fizz, //EZ bartender drink, like brainless
+							/datum/reagent/yuck) //since the other two are easy to make tons of, this is kind of a limiter
+
+	supplementary_reagents = list( //all of these are just geyser stuff, rated by their rarity
+							/datum/reagent/wittel = 10, //stupid rare
+							/datum/reagent/medicine/omnizine/protozine = 5,
+							/datum/reagent/plasma_oxide = 3,
+							/datum/reagent/clf3 = 1)//since this is also chemistry it's worth near nothing
+
+	suppressive_reagents = list(//generics you would regularly put in a vat kill abberant residue
+						/datum/reagent/consumable/nutriment/peptides = -6,
+						/datum/reagent/consumable/nutriment/protein = -4,
+						/datum/reagent/consumable/nutriment = -3,
+						/datum/reagent/liquidgibs = -2
+						)
+	virus_suspectibility = 0
+	resulting_atoms = list(/mob/living/simple_animal/hostile/netherworld = 1)
+
+/datum/micro_organism/cell_line/netherworld/succeed_growing(obj/machinery/plumbing/growing_vat/vat)
+	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the faceless is whiff.
+	resulting_atoms = list(random_result = 1)
```

Comment:
This is bothering travis for some in- TRAVIS I LOVE YOU SO FUCKING MUCH.
![image](https://user-images.githubusercontent.com/58055496/91152619-0c966880-e674-11ea-9304-827722878f9e.png)
ya can't use vars in list defs like this
```suggestion
	var/random_result = pick(typesof(/mob/living/simple_animal/hostile/netherworld)) //i looked myself, pretty much all of them are reasonably strong and somewhat on the same level. except migo is the jackpot and the faceless is whiff.
	resulting_atoms = list()
	resulting_atoms[random_result] = 1
```

Reply:
I can't believe travis has done this

Reply:
Apparently you can also do `resulting_atoms = list((random_result) = 1)`